arch/arc/core/mpu/arc_mpu_v2_internal.h

```diff
@@ -16,10 +16,6 @@
 #define AUX_MPU_RDP_ATTR_MASK (0x1FC)
 #define AUX_MPU_RDP_SIZE_MASK (0xE03)
 
-#define _ARC_V2_MPU_EN   (0x409)
```

Comment:
Cannot we move that non functional change to the separate commit (even if you prefer to include it in this PR)?